tests/simulation/params/test_validators_bet_disk.py

```diff
@@ -0,0 +1,144 @@
+import unittest
+
+import pytest
+
+import flow360.component.simulation.units as u
+from flow360.component.simulation.models.volume_models import BETDisk
+from flow360.component.simulation.simulation_params import SimulationParams
+from tests.simulation.translator.utils.xv15_bet_disk_helper import createBETDiskSteady
+from tests.simulation.translator.utils.xv15BETDisk_param_generator import (
+    _BET_cylinder,
+    _rpm_hover_mode,
+)
+
+assertions = unittest.TestCase("__init__")
+
+
+@pytest.fixture
+def create_steady_bet_disk():
+    bet_disk = createBETDiskSteady(_BET_cylinder, 10, _rpm_hover_mode)
+    return bet_disk
+
+
+@pytest.fixture(autouse=True)
+def change_test_dir(request, monkeypatch):
+    monkeypatch.chdir(request.fspath.dirname)
+
+
+@pytest.mark.usefixtures("array_equality_override")
```

Comment:
Do we need these @pytest.mark.usefixtures("array_equality_override") ?

Reply:
may not, removed.